index.js

```diff
@@ -9,7 +9,7 @@ const resolve = moduleId => {
 	} catch (_) {}
 };
 
-const clear = moduleId => {
+const clear = (moduleId, options) => {
```

Comment:
```suggestion
const clear = (moduleId, options = {}) => {
```

Reply:
Added the default parameter.  Unfortunately, the default parameter in plain javascript is fairly useless as you could still call it with `clearModule('./my-module', null)` and it will error out.  I've left the `type options === 'object'` test in there so as not to error.  The current clearModule function would not error if that was passed in.  My goal was to make it 100% backwards compatible.

Reply:
i could do a `(moduleId, opts) =>` then a `const options = opts || {}`... ?? would be better but its not available in Node 8, only in 14+.  This would at least guarantee that options is an object.

Reply:
> Unfortunately, the default parameter in plain javascript is fairly useless as you could still call it with clearModule('./my-module', null) and it will error out.

No, that's a good thing. It should error on incorrect values.

> The current clearModule function would not error if that was passed in. My goal was to make it 100% backwards compatible.

There was never a documented second argument, so there's no backwards compatibility concern.

Reply:
True for documentation, but  "1234".filter // no exception, (1234).filter // no exception... but null.filter = exception.  clearModule('./my-module', 123) // doesn't throw an exception, but considers no options set